app/assets/javascripts/application.js

```diff
@@ -15,3 +15,12 @@
 //= require turbolinks
 //= require cartify/application
 //= require_tree .
+function openDiv(cat) {
+    let i, tabCategory;
+    tabCategory = document.getElementsByClassName("tabCategory");
+    for (i = 0; i < tabCategory.length; i++) {
+        tabCategory[i].style.display = "none";
+        }
+        document.getElementById(cat).style.display = "block";
+    }
+    document.getElementsByClassName("tabs")[0].click()
```

Comment:
`tabCategory` is not a conventional class name

Please use the `forEach` method for iteration over the `tabCategory` (should be named in plural, btw)  collection. If it's a NodeList, you can use: 
```
Array.from(tabCategory).forEach(function(category){
   // whatever you want to do...
})

What and, most importantly WHEN is 
```
document.getElementsByClassName("tabs")[0].click()
````
supposed to be exectuted?